### ISSUE TITLE
PinkMan doesn't work if you remove lock screen

### ISSUE THREAD

**Issue by lestatte** [RedMadRobot/PINkman#25]
https://github.com/RedMadRobot/PINkman/pull/24 - it is only half-fix @Zestxx

I suggest you always put setUnlockedDeviceRequired(false) instead of setUnlockedDeviceRequired(isDeviceSecure)

Why:

Steps for android 12

1) set android lock screen
2) install pinkman demo app (i used fork of this repo)
3) create pin in pinkman
4) remove android lock screen
5) launch pinkman demo app

Expected result - all works
Actual result - crash

It happens, because you generate key on locked device, This key becomes irrevocably lost, if user removes android lock screen.
Generated key also may become irrevocably lost, if user for example change android lock screen type from PIN to Graphical key. Or may be, if user just changes PIN's digits.

**Comment by lestatte** [comment]
Yes we saw [this issue](https://issuetracker.google.com/issues/191391068?pli=1) month ago (we voted on it :) ). I thougt you know about this issue. But as we (android developers) all know google never hurry up with AOSP bug fixing. In most cases they completely ignore issues and do nothing. Issue has: Priority - P3 and Severity - S3. I think google will ignore it.

**Comment by lestatte** [comment]
@Fi5t In general ok, thanks, but may be it will be better to put:
```
                    val isDeviceSecure = Build.VERSION.SDK_INT < Build.VERSION_CODES.S && isDeviceSecure()
```
instead of
```
                    val isDeviceSecure = if (Build.VERSION.SDK_INT == 31) {
                        false
                    } else {
                        isDeviceSecure()
                    }
```

**Comment by osipxd** [comment]
Just released v1.3.1 with this fix. It will be published in 10-15 minutes
